Makefile

```diff
@@ -5,4 +5,7 @@ check-clean-cache:
 	golangci-lint cache clean
 
 test:
-	go test ./...
\ No newline at end of file
+	go test ./...
+
+errorutil:
```

Comment:
@mgfeller Can this be a `go generate <file>`?

Reply:
@kumarabd just had a look... no I don't think so.